src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/integration/unallocatedcases/UpdateUnallocatedCaseOffenderEventListenerTests.kt

```diff
@@ -314,4 +286,21 @@ class UpdateUnallocatedCaseOffenderEventListenerTests : IntegrationTestBase() {
 
     assertThat(case.initialAppointment).isNull()
   }
+
+  private fun publishConvictionChangedMessage(crn: String) {
```

Comment:
I think this should be part of IntegrationtestBase rather than having 2 version of it (and in fact a tier version as well)